src/components/basic/IconHeading.js

```diff
@@ -14,13 +14,13 @@ const Heading = props => {
 };
 
 const IconHeading = props => {
-  const { icon, image } = props;
+  const { icon, image, alt } = props;
```

Comment:
Instead of passing this as a prop, our images in this are decorative only so based on w3 guidlines this should be an empty alt tag.

https://www.w3.org/WAI/tutorials/images/decorative/

Reply:
Done